problem_builder/migrations/0004_copy_course_ids.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+def copy_course_id_to_course_key(apps, schema_editor):
```

Comment:
Actually, if we're just going to cut a release directly off this branch it shouldn't matter.  We can still resolve conflicts and merge the branch afterward.  I've set the migration to noop.